RecoParticleFlow/PFProducer/test/mlpf_sonic.py

```diff
@@ -0,0 +1,62 @@
+from FWCore.ParameterSet.VarParsing import VarParsing
+# Forked from SMPJ Analysis Framework
+import FWCore.ParameterSet.Config as cms
+import os, sys, json
+
+options = VarParsing("analysis")
+options.register("address", "127.0.0.1", VarParsing.multiplicity.singleton, VarParsing.varType.string)
+options.register("port", 8001, VarParsing.multiplicity.singleton, VarParsing.varType.int)
+options.register("timeout", 30, VarParsing.multiplicity.singleton, VarParsing.varType.int)
+options.register("threads", 1, VarParsing.multiplicity.singleton, VarParsing.varType.int)
+options.register("streams", 0, VarParsing.multiplicity.singleton, VarParsing.varType.int)
+options.register("batchSize", 1, VarParsing.multiplicity.singleton, VarParsing.varType.int)
+options.parseArguments()
+
+process = cms.Process('mlpftest')
+
+process.source = cms.Source("EmptySource")
```

Comment:
will this config actually run with `EmptySource`? (the toy examples use `EmptySource` because they're not actually reading event data)

Reply:
this file was committed by mistake.